notebooks/official/custom/get_started_vertex_training_xgboost.ipynb

```diff
@@ -0,0 +1,992 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Make folder for Python training script
</pre><p>nit: I'm trying to not to mix Python and bash in the same cell, since I've run into issues with bash commands not being executed correctly.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1665/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>This cell sequence has been frequently repeated without execution issues</p>